src/components/services/Linkding.vue

```diff
@@ -0,0 +1,68 @@
+<template>
+  <template v-for="bookmark in bookmarks" :key="bookmark.name">
+      <Generic :item="bookmark">
```

Comment:
Untested but should work simplified as: 
```suggestion
<template>
      <Generic v-for="bookmark in bookmarks" :key="bookmark.name" :item="bookmark">
```